stdlib/public/runtime/Errors.cpp

```diff
@@ -91,21 +91,19 @@ static bool getSymbolNameAddr(llvm::StringRef libraryName,
   // demangle with swift. We are taking advantage of __cxa_demangle actually
   // providing failure status instead of just returning the original string like
   // swift demangle.
+  static StaticMutex mutex;
```

Comment:
The `#if defined(_WIN32)` needs to come before this line, I think.

Reply:
Ah, right you are - it seems that I didn't move the macros around properly